enterprise-suite/templates/es-console.yaml

```diff
@@ -150,7 +151,7 @@ data:
       }
 
       location ~ (.*/service/alertmanager)(/.*) {
-        proxy_pass http://$alertmanager:9093$2$is_args$args;
+        proxy_pass http://$alertmanager:$alertmanagerport$2$is_args$args;
         proxy_redirect '/' ' $1/';
       }
```

Comment:
Does this make sense if the user has an external alertmanager? We could probably disable this section in that case (it's up to the user to manage access to their external AM).

Reply:
It still works. In case of external alertmanager, we need to know its address anyway, so we can setup forwarding here. That is used at least for alertmanager UI button in Console.

The only quirk I can think of is that in case of HA alertmanager cluster, this would always forward to the first alertmanager.